assessments/Renaissance_STAR/templates/assessments.jsont

```diff
@@ -45,18 +45,22 @@
       "gradeLevelDescriptor": "uri://ed-fi.org/GradeLevelDescriptor#Twelfth grade"
     },
     {
-      "gradeLevelDescriptor": "uri://ed-fi.org/GradeLevelDescriptor#No grade level"
+      "gradeLevelDescriptor": "uri://ed-fi.org/GradeLevelDescriptor#Ungraded"
     }
   ],
   "academicSubjects": [
     {% if assessment_id == 'STAR-MA' %}
     {
       "academicSubjectDescriptor": "uri://ed-fi.org/AcademicSubjectDescriptor#Mathematics"
     }
-    {% else %}
+    {% elif assessment_id == 'STAR-RD' %}
```

Comment:
Bugfix to original template. Reading and Early-Lit were merged into a single Reading subject.